src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -197,6 +209,25 @@ define([
             }
         },
 
+        _executeNanoFlow: function() {
+            if (this.nanoflow.nanoflow && this.mxcontext) {
+                mx.data.callNanoflow({
+                    nanoflow: this.nanoflow,
+                    origin: this.mxform,
+                    context: this.mxcontext,
+                    callback: lang.hitch(this, function(result) {
+                        if (!result) {
+                            logger.debug(this.id + "._executeNanoFlow callback, stopping timer");
+                            this._stopTimer();
+                        }
+                    }),
+                    error: lang.hitch(this, function(error) {
+                        mx.ui.error("An error ocurred while executing nanoflow" + error.message);
```

Comment:
Also log this error to the `logger.error` so the error is shown in the console of the runtime

Reply:
done